### ISSUE TITLE
Seeing empty tls_certs error when trying to run examples/complete

### ISSUE THREAD

**Issue by tieujason330** [terraform-aws-modules/terraform-aws-eks#2171]
## Description

Seeing this error when running `terraform apply` in `terraform-aws-eks/examples/complete`
```
╷
│ Error: Invalid index
│
│   on ../../main.tf line 211, in resource "aws_iam_openid_connect_provider" "oidc_provider":
│  211:   thumbprint_list = concat([data.tls_certificate.this[0].certificates[0].sha1_fingerprint], var.custom_oidc_thumbprints)
│     ├────────────────
│     │ data.tls_certificate.this[0].certificates is empty list of object
│
│ The given key does not identify an element in this collection value.
```
- [X] ✋ I have searched the open/closed issues and my issue is not listed.

## Versions

- Module version [Required]: 18.26.5

- Terraform version: v1.0.1

## Reproduction Code [Required]

```
cd terraform-aws-eks/examples/complete
terraform init
terraform apply # error appears
```

## Expected behavior

Expected to be able to launch EKS cluster.

## Actual behavior

Error above appears.

**Comment by tieujason330** [comment]
looks like i opened this issue at the same time as #2170. looks to be the same issue, although my scenario isn't setting any variables

**Comment by maheimstaden** [comment]
Hi!

Have the same issue - yesterday it was not there...

Kr,
Michal.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 18.26.6 :tada:
